### PR TITLE
Feat/issues from 3.11.1

### PR DESCRIPTION
**Pull request by ickisIckis**
fix(components/input-select): extra title in input select https://github.com/zyfra/Prizm/issues/1286
fix(components/input-layout-date-time): custom error not shown for inputLayoutDateTime https://github.com/zyfra/Prizm/issues/185
fix(components/input-layout): clear button in inputs with position=center has incorrect placement https://github.com/zyfra/Prizm/issues/1385
fix(doc/inputStatusText): inputStatusText examples fixed
fix(components/switcher): switcher active item disabled state colors changed https://github.com/zyfra/Prizm/issues/1348
fix(components/navigation-menu): navigation menu search error https://github.com/zyfra/Prizm/issues/1344
fix(components/navigation): navigation menu active item setter error https://github.com/zyfra/Prizm/issues/1213
fix(components/breadcrumbs): breadcrumbs force update for content projection usage https://github.com/zyfra/Prizm/issues/1282
fix(components/input-number): incorrect height of InputNumberDefaultControlsComponent https://github.com/zyfra/Prizm/issues/1308, https://github.com/zyfra/Prizm/issues/1383
fix(components/inputs): incorrect controls height https://github.com/zyfra/Prizm/issues/1179
fix(components/input-button): input button size set to 100% height of the container
fix(components/input-date-time-range): incorrect injection token for date time value transformer https://github.com/zyfra/Prizm/issues/1364
fix(components/tabs): issue with close tabs logic https://github.com/zyfra/Prizm/issues/1360
fix(doc/dropdown-host): select panel example styles fix https://github.com/zyfra/Prizm/issues/1283
fix(components/paginator): updated i18n https://github.com/zyfra/Prizm/issues/1390 https://github.com/zyfra/Prizm/pull/1391
!!! if you use custom i18n dictionary you can get bc (our [ADR](https://jira.zyfra.com/wiki/x/crs6Dw))
fix(component/tabs): error with double emitted index on close tabs https://github.com/zyfra/Prizm/pull/1392 https://github.com/zyfra/Prizm/issues/1003 https://github.com/zyfra/Prizm/pull/1389

### CODE REVIEW

**Comment by ickisIckis** [comment]
Заблокировано: иконки

File: libs/i18n/src/lib/languages/russian/russian.ts [hunk 1]
```diff
@@ -14,6 +15,7 @@ export const PRIZM_RUSSIAN_LANGUAGE = {
   ...PRIZM_RUSSIAN_LANGUAGE_KIT,
   ...PRIZM_RUSSIAN_FILE_UPLOAD,
   ...PRIZM_RUSSIAN_INPUT_LAYOUT_DATE_RELATIVE,
+  ...PRIZM_RUSSIAN_INPUT_LAYOUT_DATE_TIME,
```
Comment (ZurabDev) [inline libs/i18n/src/lib/languages/russian/russian.ts:18]: А в English словарь добавили переводы также, надо сверить на наличие всех токенов в обоих словарях

File: libs/components/src/lib/components/tabs/tabs.service.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { Injectable, OnDestroy } from '@angular/core';
 import { BehaviorSubject, combineLatest, concat, Observable, of, Subject } from 'rxjs';
-import { distinctUntilChanged, filter, map, startWith, take, takeUntil, tap } from 'rxjs/operators';
+import { debounceTime, distinctUntilChanged, map, take, takeUntil, tap } from 'rxjs/operators';
```
Comment (ZurabDev) [inline libs/components/src/lib/components/tabs/tabs.service.ts:3]: Они используются?

File: libs/components/src/lib/components/tabs/tabs.module.ts [hunk 3]
```diff
@@ -3,6 +3,9 @@ import { PrizmTabComponent } from './components/tab.component';
 import { PrizmTabsComponent } from './tabs.component';
 import { PrizmTabMenuItemDirective } from './tab-menu-item.directive';
 
+/**
+ * use standalone
```
Comment (ZurabDev) [inline libs/components/src/lib/components/tabs/tabs.module.ts:7]: Нужно ли тут этот призыв, если модуль экспортирует множ сущностей?

**Review by ZurabDev — APPROVED** [review]
LGTM

**Review by alexhawkins94 — APPROVED** [review]

**Pull request merged**